### DISCUSSION TITLE
[BP#86] Choisir un hébergeur "vert"

### DISCUSSION THREAD

**Discussion by DocRoms** [cnumr/best-practices#38] [Bonnes pratiques] [answered]
La règle [BP#86](https://github.com/cnumr/best-practices/blob/main/chapters/BP_086_fr.md) stipule : 

----
**Règle**
Préférer un hébergeur qui combine des serveurs économes à des sources d’énergies renouvelables et qui compense ses émissions de gaz à effet de serre. Les six critères à prendre en compte sont :

gestion des DEEE (déchets d’équipements électriques et électro- niques) ;
efficience énergétique du data center ;
politique d’achat responsable ;
respect de la dimension sociale ;
alimentation aux énergies renouvelables ;
compensation carbone.

**Exemple**
Les hébergeurs suivants utilisent exclusivement des énergies renouvelables ou compensées :
www.aiso.net
www.infomaniak.com

----

Je penses que la référence aux `énergies renouvelables` devrait être remplacée par `énergies bas carbone`. En effet, énergétiquement, nous avons besoins de **toutes** les sources d'énergies bas carbone possible, et pas juste aux ENRIs, qui possèdent en général (et actuellement) d'un backup augmentant leur GES sur leur ACV, quand elles ne produisent pas ;)

**Comment by ACTLEM** [comment]
Tu as raison, cette BP mérite d'être revue. Personnellement, je suis surtout mal à l'aise avec "la compensation carbone" car en général cela se traduit par "payer pour polluer". Par exemple, la neutralité affichée cache le fait que l'empreinte augmente.
Pour être plus clair, alors qu'une entreprise (ou datacenter) augmente ses émissions de gaz à effet de serre, elle peut afficher une compensation carbone via en général des projets de reforestation. Or, des projets de reforestation ne fonctionnent pas du jour au lendemain, de plus certains partent en fumée, et plus il y a de gaz à effet de serre moins les forêts seront efficaces dans leur absorption.

Je suggère donc de garder la gestion DEEE, l'efficience énergétique, le respect de la dimension sociale, la politique d'achat responsable (même si pour ce dernier cela mérite des précisions).
J'ajouterais "une alimentation sans énergie fossile (pétrole, gaz, charbon)" et une "transparence sur les résultats de réductions des impacts environnementaux (GES, eau, ressources abiotiques, déforestation, etc...)"

**Reply by Trankilloman** [reply]
En effet, c'est très dangereux de parler de compensation carbone (car souvent que en lien avec la conso d'énergie, pas sur l'ensemble du cycle de vie et surtout c'est du monocritere et donc c'est un coup à générer beaucoup de transferts d'impacts).
Il serait aussi possible de préconiser de choisir des fournisseurs ayant des démarches d'améliorations environnementales normés ou réglementées (iso14001:2015, ecolabel officiel...).

**Reply by Trankilloman** [reply]
Le terme "vert" est problématique également, j'aurais mis responsable à la place.

**Reply by Christophe-Cl** [reply]
"éco-responsable" ? si on reste sur l'écoconception, donc le facteur environnemental

**Comment by arnolem** [comment]
Au lieu de ne citer que 2 hébergeurs, pourquoi ne pas renvoyer vers une ressource actualisée qui listerait des hébergeurs éco-responsable ?
Voici un exemple mais il en existe surement d'autre : https://www.thegreenwebfoundation.org/directory/#FR

**Reply by Fred-artwai** [reply]
Il y a même une API pour savoir si son site est hébergé plus respectueusement que les autres.

**Comment by tbroyer** [comment]
À titre personnel, je refuse de cautionner un hébergeur qui finance le CRIIGEN ou Kokopelli: https://www.infomaniak.com/fr/hebergeur-ecologique/mecenat-associations

Plus objectivement, je pense que le mieux serait encore de ne **pas** orienter le lecteur vers des hébergeurs spécifiques, ou même vers une liste dont on ne connait pas vraiment l'objectivité, la représentativité, ou les limites ; en supprimant tout bonnement les exemples.

Si on pouvait par contre donner quelques indications concernant les indicateurs listés sur des valeurs "acceptables". Par exemple, OVH et Scaleway sont assez transparents (https://corporate.ovhcloud.com/fr/sustainability/environment/, https://www.scaleway.com/fr/leadership-environnemental/), mais je ne saurais pas dire si ces valeurs sont bonnes ou non. Ça me permet de comparer avec d'autres hébergeurs, mais savoir s'ils sont "bons" dans l'absolu, et pas seulement relativement les uns aux autres serait un plus.

**Reply by florinesueur** [reply]
Il me semble si nous voulions recommander des hébergeurs, il faudrait se lancer dans un benchmark pour avoir accès aux informations comme cité par @tbroyer et [https://pue.dc3.scaleway.com/fr/ ](https://pue.dc3.scaleway.com/fr/) qui ne sont pas intemporelles. C'est un travail de titan ! 

Personnellement, je préfère qu'on m'apprenne que le PUE, c'est : "Le PUE (Power Usage Effectiveness) est une unité de mesure développée par 'The Green Grid' pour mesurer l'efficacité d'un datacenter, et normalisé par la norme ISO/IEC 30134-2:2016.
Il s'agit du ratio entre l'énergie totale utilisée par l'ensemble du bâtiment et l'énergie délivrée aux serveurs informatiques.
Le PUE idéal est de 1,0 ce qui signifie que 100 % de l'énergie du datacenter est utilisée par les serveurs informatiques. Actuellement, les meilleures pratiques du secteur permettent d'atteindre un PUE de 1,5." Donc un PUE de 1.5, c'est super bien.

Finalement, comment checker les critères et comment (sur quelles données) les hébergeurs sont-ils comparables ?

**Reply by Christophe-Cl** [reply]
PUE ++ et aussi le CUE (Carbon Usage Effectiveness) et le WUE (Water Usage Effectiveness) qui sont des indicateurs importants dans le choix d'un hébergeur, si celui-ci les affiche.

**Comment by ACTLEM** [comment, marked as answer]
https://github.com/cnumr/best-practices/issues/144
